website/docs/index.html.markdown

```diff
@@ -1,209 +1,229 @@
----
```

Comment:
Can you remove this file from the PR?

I will probably be revising the provider index doc soon, and at that point it will contain a more terse example, something more akin to the [AWS provider's index page](https://www.terraform.io/docs/providers/aws/index.html).